EXM_104/Makefile

```diff
@@ -6,6 +6,8 @@ info:
 	$(info `make` will perform patient generation and calculation.)
 
 PATIENT_COUNT := 10
+MP_START := 2020-01-01
+MP_END := 2020-12-31
 
 generate-patients-stu3:
 	cd ../synthea && ./run_synthea --exporter.fhir.export=false --exporter.fhir_stu3.export=true --exporter.baseDirectory=../EXM_104/$(SYNTHEA_DIR)/ -a 18-21 -p $(PATIENT_COUNT) -m stroke_exm_104*
```

Comment:
These are missing the passed in year option. Or is it not needed for this module?

Reply:
ah. not needed for this module in my runs that I did since it loops over multiple years. should get some patients for any year

Reply:
oh, but still needed to be passed into the fhir bundle calculator command